pycheribuild/config/target_info.py

```diff
@@ -562,11 +562,6 @@ def pkgconfig_dirs(self) -> str:
     def get_rootfs_target(self) -> "Project":
         from ..projects.cross.cheribsd import BuildCHERIBSD
         xtarget = CompilationTargets.NONE
-        # Install the purecap targets into the hybrid rootfs:
-        if self.target.is_cheri_purecap([CPUArchitecture.MIPS64]):
-            xtarget = CompilationTargets.CHERIBSD_MIPS_HYBRID
-        elif self.target.is_cheri_purecap([CPUArchitecture.RISCV64]):
-            xtarget = CompilationTargets.CHERIBSD_RISCV_HYBRID
         return BuildCHERIBSD.get_instance(self.project, cross_target=xtarget)
```

Comment:
Without this the programs don't end up in the pure cap disk image.

Reply:
Since we are moving away from using hybrid in favour of rootfs, I think this is fine.
It was required back when the full purecap userspace wasn't stable: I used a hybrid world to test new purecap projects.
I tend to mount directories via smbfs rather than embedding in the rootfs, so installing both hybrid and purecap to the same rootfs shouldn't be required anymore.

Reply:
Thinking this over (at 4am, thanks brain) I think we should probably make all the -purecap cross targets default to using the purecap sdk and rootfs, but perhaps should retain a fallback to using the hybrid version.